### PR TITLE
Improve REX pull

### PR DESCRIPTION
**Pull request by Lennonka**
https://bugzilla.redhat.com/show_bug.cgi?id=2136485

* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.4/Katello 4.6
* [x] Foreman 3.3/Katello 4.5
* [ ] Foreman 3.2/Katello 4.4
* [ ] Foreman 3.1/Katello 4.3

### CODE REVIEW

File: guides/common/modules/proc_migrating-from-katello-agent-to-remote-execution.adoc [hunk 1]
```diff
@@ -15,12 +15,12 @@ endif::[]
 * You have previously installed the `katello-agent` package on content hosts.
 
 .Procedure
-. If you have Remote Execution configured to use `pull-mqtt` mode: Deploy the remote execution pull client to the hosts.
-ifdef::katello[]
+. If you have Remote Execution configured to use `ssh` mode, distribute the remote execution SSH keys to the hosts.
+For more information, see xref:Distributing_SSH_Keys_for_Remote_Execution_{context}[].
+. If you have Remote Execution configured to use `pull-mqtt` mode, deploy the remote execution pull client to the hosts.
```
Comment (jeremylenz) [inline guides/common/modules/proc_migrating-from-katello-agent-to-remote-execution.adoc:20]: Shouldn't we have a link here to the steps to do so?

**Review by jeremylenz — COMMENTED** [review]
Thanks @Lennonka 

Looks good except one comment below

File: guides/common/modules/proc_migrating-from-katello-agent-to-remote-execution.adoc (hunk 1 above)
Comment (Lennonka) [reply guides/common/modules/proc_migrating-from-katello-agent-to-remote-execution.adoc:20]: Yes, the link is located inside the ifdef below. The ifdef was there only for katello, but it should have been there for satellite and orcharhino as well, which I've corrected in this PR.

**Review by adamlazik1 — APPROVED** [review]
LGTM from wording perspective.

File: guides/common/modules/proc_migrating-from-katello-agent-to-remote-execution.adoc (hunk 1 above)
Comment (jeremylenz) [reply guides/common/modules/proc_migrating-from-katello-agent-to-remote-execution.adoc:20]: aah, gotcha!

**Review by jeremylenz — APPROVED** [review]
My single concern is addressed. LGTM! :+1:

**Comment by Lennonka** [comment]
Cherry-picked:
- c19a8e9da..778a608b0  3.4 -> 3.4
- 46df46700..7b10e111f  3.3 -> 3.3

**Pull request merged**